lib/sidekiq_unique_jobs/orphans/manager.rb

```diff
@@ -177,7 +179,7 @@ def enabled?
       # @return [void]
       #
       def register_reaper_process
-        redis { |conn| conn.set(UNIQUE_REAPER, current_timestamp, nx: true, ex: drift_reaper_interval) }
+        redis { |conn| conn.set(UNIQUE_REAPER, current_timestamp, ex: drift_reaper_interval * EXPIRATION_FACTOR) }
```

Comment:
1. I mistakenly thought that if a new reaper process starts and the key is not updated, resurrector will consider the reaper dead and start a new process. However, I see, that the task is executed right after the process starts and the mutex will be updated. I will revert this change.
2. I faced an issue, when the key expired before the mutex was updated and a duplicate reaping process started. Most likely this happened because I was testing the functionality on low intervals (around 10 seconds) and the `DRIFT_FACTOR` did not increase the interval anyhow. I will retest this on higher intervals.